src/commands/Games/higherLower.ts

```diff
@@ -7,6 +7,7 @@ import { LLRCData, LongLivingReactionCollector } from '@utils/LongLivingReaction
 import { resolveEmoji } from '@utils/util';
 import { MessageEmbed } from 'discord.js';
 import { KlasaMessage } from 'klasa';
+import { UserEntity } from '@orm/entities/UserEntity';
```

Comment:
```suggestion
import { UserEntity } from '@orm/entities/UserEntity';
import { resolveEmoji } from '@utils/util';
import { MessageEmbed } from 'discord.js';
import { KlasaMessage } from 'klasa';
```

One day I'll get you to enable organize imports on save and format on save and you'll use them. One day...